lib/index.js

```diff
@@ -90,36 +90,37 @@ function managedError(err) {
     var error, msg;
     if (err.name === 'TimeoutError') {
         error = new Error('Timed out after 5000ms');
-        error.status = 504;
+        error.statusCode = 504;
         error.retryable = true;
     } else if (err.statusCode === 404 || err.statusCode === 403) {
         error = new Error('Tile does not exist');
-        error.status = err.statusCode;
+        error.statusCode = err.statusCode;
         error.retryable = false;
     } else if (err.code === 'NetworkingError' && err.message === 'socket hang up') {
         error = new Error(err.message);
-        error.status = 500;
+        error.statusCode = 500;
         error.code = 'ECONNRESET';
         error.retryable = true;
     } else if (err.code === 'InternalError') {
         error = new Error(err.message);
-        error.status = 500;
+        error.statusCode = 500;
         error.code = 'InternalError';
         error.retryable = true;
     } else if (err.code === 'TruncatedResponseError') {
         // This error is generated by tilelive-s3 -- it is not recognized as an
         // error by aws-sdk. See S3.get below.
         error = new Error('Content-Length does not match response body length');
-        error.status = 500;
+        error.statusCode = 500;
         error.code = 'TruncatedResponseError',
         error.retryable = true;
     } else if (err.statusCode === 503 && !err.message) {
         error = new Error('503 Unknown');
-        error.status = 503;
+        error.statusCode = 503;
         error.retryable = true;
     } else if (err.statusCode && !err.status) {
         error = err;
-        error.status = err.statusCode;
+        error.statusCode = err.statusCode;
+        error.retryable = false;
```

Comment:
This change is somewhat of a mystery to me -- it seems like it should have been here in `master` as well. I need to look at this harder on `master` to understand why it's not needed. cc @rclark